packages/scripts/config/webpack.config.js

```diff
@@ -2,13 +2,16 @@
  * External dependencies
  */
 const { BundleAnalyzerPlugin } = require( 'webpack-bundle-analyzer' );
+const IgnoreEmitPlugin = require( 'ignore-emit-webpack-plugin' );
 const LiveReloadPlugin = require( 'webpack-livereload-plugin' );
+const MiniCssExtractPlugin = require( 'mini-css-extract-plugin' );
```

Comment:
CSS is an acronym.

https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#abbreviations-and-acronyms

```suggestion
const MiniCSSExtractPlugin = require( 'mini-css-extract-plugin' );
```

Reply:
Updated in 46c2f17.